ci-operator/config/openshift-priv/aws-pod-identity-webhook/openshift-priv-aws-pod-identity-webhook-release-4.5.yaml

```diff
@@ -13,10 +13,6 @@ canonical_go_repository: github.com/aws/amazon-eks-pod-identity-webhook
 images:
 - dockerfile_path: Dockerfile.rhel7
   from: base
-  inputs:
-    bin:
-      as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
```

Comment:
?

https://github.com/openshift/aws-pod-identity-webhook/blob/2251f7e2fc9194aa3a57cd9dc1de4a71e9367df5/Dockerfile.rhel7#L1

Reply:
yeah because it is priv. Fixed in https://github.com/openshift/ci-tools/pull/997

Reply:
Can we run with a privileged token?

Reply:
Yes, but lets do that in a follow-up.